deployment/user_data.sh.tpl

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Specify the log file
+LOG_FILE="/var/log/user_data.log"
+
+# Sleep for 60 seconds
+echo "Sleeping for 60 seconds before executing commands..."
+sleep 60
+
+# Use exec to redirect all stdout and stderr to the log file from this point on
+exec > >(tee -a ${LOG_FILE}) 2>&1
+
+echo "Starting user_data script execution"
+
+# No need for 'sudo su' when running as user data, the script runs with root privileges by default
```

Comment:
Here you wrote no need to write sudo but all the commands below are using sudo :P